cli/tests/integrations/test_package.py

```diff
@@ -75,8 +75,14 @@ def test_repo_list():
         ).format(**UNIVERSE_TEST_REPOS),
         'utf-8'
     )
+
     assert_command(['dcos', 'package', 'repo', 'list'], stdout=repo_list)
 
+    # test again, but override the dcos_url with a cosmos_url config
+    dcos_url = config.get_config_val("core.dcos_url")
+    with update_config('package.cosmos_url', dcos_url, env):
+        assert_command(['dcos', 'package', 'repo', 'list'], stdout=repo_list)
+
```

Comment:
this test makes it clear that if the second part of the test fails it's because of the cosmos_url. if there were to be a separate test, it would be unclear if the cosmos_url part didn't work or the output was wrong for another reason and/or the ordering of the tests would matter.